spec/features/invoices/index_spec.rb

```diff
@@ -46,13 +44,73 @@
 
     it "overdue invoice has class 'overdue'" do
       within '#invoices_list' do
-        expect(subject).to have_css("tr.invoice.overdue##{dom_id(overdue_invoice)}")
+        expect(page).to have_css("tr.invoice.overdue##{dom_id(overdue_invoice)}")
       end
     end
 
     it "paid invoice has class 'paid'" do
       within '#invoices_list' do
-        expect(subject).to have_css("tr.invoice.paid##{dom_id(paid_invoice)}")
+        expect(page).to have_css("tr.invoice.paid##{dom_id(paid_invoice)}")
+      end
+    end
+  end
+
+  context 'complete invoice', js: true do
+    let!(:account)  { create :bank_account, organization: org }
+    let!(:category) { create :category, :income, organization: org }
+    let!(:wrong_category) { create :category, :expense, organization: org }
+    let!(:wrong_account)  { create :bank_account, organization: org, currency: 'USD' }
+    let!(:invoice)  { create :invoice, organization: org, amount: 500 }
+    let(:comission) { Money.new(100, invoice.currency) }
+
+    def create_transaction_by_invoice
+      visit invoices_path
+      click_on 'Complete Invoice'
+      within '#new_transaction' do
+        select category.name, from: 'transaction[category_id]'
+        select account.name, from: 'transaction[bank_account_id]'
+        fill_in 'transaction[comission]', with: comission
+        fill_in 'transaction[comment]', with: 'TestComment'
+      end
+      expect(page).to_not have_select('Category', with_options: [wrong_category.name])
+      expect(page).to_not have_select('Bank Account', with_options: [wrong_account.name])
+      expect(page).to have_field('Comission', with: comission)
+      expect(page).to have_content("Total amount: #{invoice.amount - comission}" )
+      click_on 'Create'
+      sleep 1 # wait after page rerender
+    end
+
+    subject{ create_transaction_by_invoice; page }
+
+    context 'update invoice paid_at after create transaction' do
+      before do
+        create_transaction_by_invoice
+        visit invoices_path
+      end
+
+      it 'invoice paid_at must present' do
+        expect(page).to have_content(invoice.paid_at)
```

Comment:
@yunixon please check exact row and exact column in the table with this content. There might be another date on the date, that just match with paid_at.